backend/src/api/organization/organizationFind.ts

```diff
@@ -20,7 +22,19 @@ import PermissionChecker from '../../services/user/permissionChecker'
 export default async (req, res) => {
   new PermissionChecker(req).validateHas(Permissions.values.organizationRead)
 
-  const payload = await new OrganizationService(req).findById(req.params.id)
+  const segmentId = req.query.segmentId
```

Comment:
Is it some new functionality? 
Originally, segments are coming through the query parameter called `segments`, and there might be many of them.
If you want to get segments coming with the request, you're looking for `req.currentSegments`.
If you want just one, then what you need is `SequelizeRepository.getStrictlySingleActiveSegment(req)`. But keep in mind, this particular request might accept multiple segments.

Reply:
Also because of how OpenSearch is indexed for other query endpoints we can always provide just one segment id and it can be either:
- 1 leaf segment id
- 1 parent segment id
- 1 grandparent segment id
because we have documents for each leaf segment id and then one aggregated for the parent segment id and also one aggregated for grandparent segment id.

This is why this logic is similar - we get only one segmentId as a parameter - matching the logic from all other relevant endpoints.